dockers/docker-base/Dockerfile.j2

```diff
@@ -1,6 +1,6 @@
-{% if CONFIGURED_ARCH == "armhf" and MULTIARCH_QEMU_ENVIRON == "y" %}
```

Comment:
>MULTIARCH_QEMU_ENVIRON [](http://example.com/codeflow?start=37&length=22)

If removed, do we remove support on any platform?

Reply:
For armhf, we can always use multiarch/debian-debootstrap:armhf-jessie, it should not be relative to MULTIARCH_QEMU_ENVIRON yes or not.